qmq-common/src/test/java/qunar/tc/qmq/utils/utils/RetrySubjectUtilsTest.java

```diff
@@ -0,0 +1,42 @@
+package qunar.tc.qmq.utils.utils;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+import static qunar.tc.qmq.utils.RetrySubjectUtils.*;
+
+public class RetrySubjectUtilsTest {
+
+    @Test
+    public void testGetRealSubject() {
+        assertNull(getRealSubject(null));
+
+        assertEquals("%RETRY", getRealSubject("%RETRY"));
+        assertEquals("foo", getRealSubject("%DEAD_RETRY%foo%bar"));
+    }
+
+    @Test
+    public void testParseSubjectAndGroup() {
+        assertNull(parseSubjectAndGroup(null));
+        assertNull(parseSubjectAndGroup("%DEAD_RETRY"));
+
+        assertNotNull(parseSubjectAndGroup("%DEAD_RETRY%foo%bar"));
+    }
+
+    @Test
+    public void testIsRealSubject() {
+        assertFalse(isRealSubject(null));
+        assertFalse(isRealSubject("%RETRY"));
+        assertFalse(isRealSubject("%DEAD_RETRY"));
```

Comment:
Are you not able to create True case in this scenario?

Reply:
I added three assertTrue cases

> What percentage of this class have you covered?
100% for this class